db/index.js

```diff
@@ -1,11 +1,10 @@
 // ℹ️ package responsible to make the connection with mongodb
 // https://www.npmjs.com/package/mongoose
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 
 // ℹ️ Sets the MongoDB URI for our app to have access to it.
 // If no env has been set, we dynamically set it to whatever the folder name was upon the creation of the app
-
-const MONGO_URI = process.env.MONGODB_URI || "mongodb://127.0.0.1/hangout-app";
+const MONGO_URI = 'mongodb://127.0.0.1/hangout-app' || process.env.MONGODB_URI;
```

Comment:
@Val-be this is going to cause a bug – it will only ever use the left-hand side because it's truthy.